diagnostic_feedback/tests/unit/__init__.py

```diff
@@ -0,0 +1 @@
+__author__ = 'super'
```

Comment:
What does this mean (super)? And why do we care about the author of an empty file anyways?